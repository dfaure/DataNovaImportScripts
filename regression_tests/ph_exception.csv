00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-04-22;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-03-19;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-04-01;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-03-20;FERME;;;;;;;;;;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-04-20;08:00-12:00;13:30-16:30;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-03-21;FERME;;;;;;;;;;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-03-09;08:00-12:00;13:30-16:30;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-03-11;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-02-12;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-03-02;08:00-12:00;13:30-16:30;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-03-08;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-03-07;FERME;;;;;;;;;;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-03-06;FERME;;;;;;;;;;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-04-13;08:00-12:00;13:30-16:30;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-03-04;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-03-10;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-02-09;08:00-12:00;13:30-16:30;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-02-11;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-02-13;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-04-27;08:00-12:00;13:30-16:30;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-04-26;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-02-25;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-02-21;FERME;;;;;;;;;;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-04-23;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-03-17;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-02-23;08:00-12:00;13:30-16:30;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-02-17;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-02-16;08:00-12:00;13:30-16:30;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-03-13;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-03-16;08:00-12:00;13:30-16:30;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-05-02;FERME;;;;;;;;;;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-04-02;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-02-14;FERME;;;;;;;;;;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-03-25;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-02-24;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-02-28;FERME;;;;;;;;;;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-02-27;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-04-30;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-03-01;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-05-08;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-02-15;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-03-27;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-04-06;08:00-12:00;13:30-16:30;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-03-23;08:00-12:00;13:30-16:30;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-04-04;FERME;;;;;;;;;;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-03-05;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-03-30;08:00-12:00;13:30-16:30;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-04-03;FERME;;;;;;;;;;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-03-03;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-03-12;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-03-26;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-04-18;FERME;;;;;;;;;;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-04-09;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-04-10;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-03-29;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-03-28;FERME;;;;;;;;;;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-03-22;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-04-05;FERME;;;;;;;;;;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-03-18;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-04-07;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-05-01;FERME;;;;;;;;;;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-05-03;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-04-11;FERME;;;;;;;;;;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-04-08;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-04-12;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-03-31;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-04-15;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-03-15;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-03-14;FERME;;;;;;;;;;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-05-04;08:00-12:00;13:30-16:30;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-02-20;FERME;;;;;;;;;;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-04-19;FERME;;;;;;;;;;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-02-10;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-02-22;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-04-16;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-04-14;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-04-21;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-02-19;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-04-17;FERME;;;;;;;;;;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-02-26;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-05-06;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-05-09;FERME;;;;;;;;;;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-02-08;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-02-18;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-05-05;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-04-29;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-05-07;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-04-28;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-03-24;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-05-10;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-04-24;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-05-12;08:00-12:00;;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-05-13;FERME;;;;;;;;;;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-05-11;08:00-12:00;13:30-16:30;;;;;;10:30;10:30;10:30;;FERME;;;;;;;
00995A;VALLEES D ANTRAIGUES-ASPERJOC AP;2021-04-25;FERME;;;;;;;;;;;FERME;;;;;;;
