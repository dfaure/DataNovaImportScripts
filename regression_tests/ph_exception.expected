00995A|VALLEES D ANTRAIGUES-ASPERJOC AP|Mo,We-Fr 08:00-12:00; Tu 08:00-12:00,13:30-16:30; week 02-53/2 Sa 08:00-12:00; PH off; 2021 May 08 08:00-12:00; 2021 Apr 19 off
